### PR TITLE
SIE-169 Improve Test Coverage for Detect Masking

### PR DESCRIPTION
**Pull request by liuhantao9**
What's in this PR?

- Improve test coverage for face_detection_masking 

How to review?

- `pytest --cov-config=.coveragerc --cov-report term-missing --cov=main test_face_detection_masking.py test.py`

Result

- Test coverage improves to 91%

### CODE REVIEW

File: face_detection_masking/test.py [hunk 1]
```diff
@@ -33,3 +33,53 @@ def test_upload_processed_images():
 
     # If all assertions passed, then return test pass
     return "Test passed"
```
Comment (quanganhhoang) [inline face_detection_masking/test.py:35]: QQ: do we have to `return "Test passed"`?

File: face_detection_masking/test.py [hunk 2]
```diff
@@ -33,3 +33,53 @@ def test_upload_processed_images():
...
+    """
+    bucket_name = "sie-face-detection-testing-assets"
+    source_blob_name = "face_neutral.jpg"
+    destination_file_name = "test_assets/test_upload_images/test_download.jpg"
+
+    main.download_image(bucket_name, source_blob_name, destination_file_name)
+
+    assert os.path.isfile(destination_file_name) is True
+
+    os.remove(destination_file_name)
```
Comment (quanganhhoang) [inline face_detection_masking/test.py]: you might want to have this in a try catch in case `destination_file_name` doesn't exist

**Review by quanganhhoang — APPROVED** [review]
I left a few comments but approved it!

File: face_detection_masking/test.py (hunk 1 above)
Comment (liuhantao9) [reply face_detection_masking/test.py:35]: No, you don't have to. I just want to make it responsive.

File: face_detection_masking/test.py (hunk 2 above)
Comment (liuhantao9) [reply face_detection_masking/test.py]: Good point!

**Pull request merged**